src/web/components/Core/MultiSelectDropdown.scss

```diff
@@ -0,0 +1,52 @@
+.multi-select-dropdown {
+  & > button {
+    border-radius: 16px;
+    border: 1px solid #0300a5;
```

Comment:
I notice this colour matches `--theme-radio-background`. Would it be appropriate to use that variable instead? Or introduce a new one?

Reply:
good catch, I totally forgot this! Updated